lib/fizzy/CMakeLists.txt

```diff
@@ -0,0 +1,10 @@
+
+add_library(fizzy)
+add_library(fizzy::fizzy ALIAS fizzy)
```

Comment:
It is best practice to also define `fizzy::fizzy` alias. It is better to use the alias when specifying link dependencies (the `::` indicates that it is a CMake target, the name `fizzy` can also mean a system lib - no CMake error then).
This is also useful when including fizzy as a CMake subproject.